### PR TITLE
Adds Forsaken Employee

### PR DESCRIPTION
**Pull request by tile4**
## About The Pull Request
Adds Forsaken Employee, a new TETH abnormality from Limbus that has a breach that will cause a power surge and turn off all the lights in a random hallway or department room on breach for a period of time. The breach also removes some PE from Quota.

## Why It's Good For The Game
More abnormalities

## Changelog
:cl:
add: Added Forsaken Employee
imageadd: added Forsaken Employee images
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Willing to try this, maybe we can work something more into it.

**Comment by InsightfulParasite** [comment]
Test Merged 8/26/24

**Comment by Coxswain-Navigator** [comment]
Mechanically speaking this is annoying, if sound. I think destroying all of the lights is a bit much for something of such a low risk level. It would be ideal if it instead turned off the lights for a few minutes. Sometimes there just aren't clerks willing to fix everything up. My only other concern is that it is a bit boring. There is no sound effect or really any other indicator of what it is doing besides some flavor text.

**Comment by Kitsunemitsu** [comment]
I did recommend a rework to do with PE; talking with them about it

**Comment by Kitsunemitsu** [comment]
Please fix your comments.

File: code/game/objects/items/ego_weapons/teth.dm
Comment (Coxswain-Navigator) [inline code/game/objects/items/ego_weapons/teth.dm:1]: This ego weapon doesn't seem all that practical at all. At the beginning of a round (when TETH is used by agents) it is practically useless, at the end (when quota is met) It's just an amazing weapon... for clerks.  Why?

**Review by Coxswain-Navigator — CHANGES_REQUESTED** [review]

File: code/game/objects/items/ego_weapons/teth.dm
Comment (tile4) [reply code/game/objects/items/ego_weapons/teth.dm:1]: God, I don't even know what the hell I was thinking when I made that formula... Hopefully this one would be more reasonable.

**Comment by tile4** [comment]
This PR got screwed thanks to a mistake I made when rebasing so I readded the stuff into #2457 to start with a fresh slate I guess.

**Pull request closed without merging**